.github/workflows/ci.yml

```diff
@@ -26,11 +26,9 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Install Rust
-      uses: actions-rs/toolchain@v1
+      uses: dtolnay/rust-toolchain@master
```

Comment:
why pin on master?

Reply:
It's the recommended approach when passing the `toolchain` as a parameter: https://github.com/dtolnay/rust-toolchain#toolchain-expressions

(It otherwise supports using tags to select MSRV)

Reply:
As tarcieri already said, that is the recommended approach - and the only supported approach for that. Also see https://github.com/dtolnay/rust-toolchain/issues/45#issuecomment-1336100499:

> The `v1` tag only exists so that the action gets a page on https://github.com/marketplace/actions. (And it has to be a tag for that to happen.) [...] `master` and all the other branches (`stable`, `1.65` etc) are the only supported and documented way to use the action.